### DISCUSSION TITLE
Development Environment Documentation

### DISCUSSION THREAD

**Discussion by saywebsolutions** [nextcloud/bookmarks#2035] [Ideas]
Hi @marcelklehr 

May I assist with writing some development documentation to try to make it a bit easier for people get up and running with a local development environment?

I was wondering if you have any tips / strategies / wisdom about developing this app. Off the top of my head I was thinking about starting by using the nextcloud docker image and finding a way to automate installation of this git repo in the test Nextcloud environment, adding a user, and maybe creating a test data set that can be imported on launch of the dev environment?

Just wondering if you have any thoughts here, and if you think such work would be helpful for the project. Thank you!

**Comment by marcelklehr** [comment]
Hey @saywebsolutions 

adding some docs aiding in development would be great, I think. Personally, I use https://github.com/juliushaertl/nextcloud-docker-dev.
